src/harness/reference_models/fss_purge/fss_purge.py

```diff
@@ -0,0 +1,310 @@
+#    Copyright 2018 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+"""
+==================================================================================
+    This is a subset of the Pre-IAP reference model which implements
+    the FSS purge list algorithm described in R2-SGN-29.
+    The main function is fssPurgeModel().
+==================================================================================
+"""
+import numpy as np
+from reference_models.antenna import antenna
+from reference_models.interference import interference
+from reference_models.geo import vincenty
+from reference_models.propagation import wf_itm
+
+# Definition of constants and data types for FSS purge model.
+# Reference Channel bandwidth for FSS OOBE calculation
+REF_BW = 5.e6
+
+# OOBE Margin
+FSS_OOBE_MARGIN = 1

```

Comment:
Shouldn't the Margin value be moved out into a configuration file rather than setting it here

Reply:
In an earlier comment Kate   mentioned these margin values will not be configurable. Hence they are   hardcoded in the reference model.

Reply:
Will it be part of the input configuration for the test cases, meaning will it be possible for the margin value to be set for a given test case and changed for another input configuration? If that is true then it will be helpful if they are configured from outside

Reply:
Agreed with your suggestion. But Kate has a different opinion on this aiming to simplify the test cases avoiding operation time mis-configuration caused errors.

Reply:
It seems like now is the right time for me to chime in on this :) WinnForum's position is that the margin values are not configurable within the test infrastructure. The margin values are specified via a steering committee policy document [1] and the test code should use exactly those values. I strongly support the idea of having a figurative #define at the top of this file, but I don't think we should go any further than that. (And to be clear, my concern is less about mis-configuration potential and more a matter of matching the WF specs and policies.)

[1] https://workspace.winnforum.org/higherlogic/ws/public/download/5676/WINNF-SSC-0007-V1.0.1%20-%20WinnForum%20CBRS%20Protection%20Margin%20Policy.pdf

Reply:
Thanks for clarifying. I only remembered a portion and the least important portion of the reasons :)

Reply:
Thanks for clarifying this Kate